tmcprototype/new_configure_logging.py

```diff
@@ -0,0 +1,82 @@
+import sys
+import json
+import time
+import numpy as np
+from tango import DeviceProxy, DevFailed
+
+# Update file path to devices.json in order to test locally
+# To test on docker environment use path : /app/tmcprototoype/devices.json
+config_json_file = '/app/data/tmc-devices.json'
+with open(config_json_file, 'r') as file:
+    str_json_devices = file.read().replace('\n', '')
+
+# Creating SKALogger DeviceProxy
+logger_device = "ref/elt/logger"
+logger_proxy = DeviceProxy(logger_device)
+print("Logger_proxy is :",logger_proxy)
+
+
+max_retries = 10
+retry = 0
+while retry < max_retries:
+    try:
+        logger_proxy.ping()
+        break
+    except DevFailed:
+        time.sleep(3)
+        retry += 1
+
+if(retry < max_retries):
+    logging_target = "device::" + logger_device
+    print(" loggging target is",logging_target)
+
+    try:
+        # Parse json
+        json_devices = json.loads(str_json_devices)
+        device_servers = json_devices["servers"]
+
+        for device_server in device_servers:
+            try:
+
+                if (device_server != "SKALogger" and
+                    device_server != "TangoTest" and
+                    device_server != "DataBase" and
+                    device_server != "TangoAccessControl"
+                ):
+                    server_instance_list = device_servers[device_server].keys()
+                    #print("server_instance_list is", server_instance_list)
```

Comment:
Add comments wherever needed. Also remove the unnecessary print statements.

Reply:
Removed unnecessary print statements.

Reply:
Commit ID: d8488c155e50d5c4940dbe52585ac2446c43334a